python/ray/autoscaler/updater.py

```diff
@@ -165,9 +165,10 @@ def wait_for_ssh(self, deadline):
                 logger.debug("NodeUpdater: "
                              "{}: Waiting for SSH...".format(self.node_id))
 
-                with open("/dev/null", "w") as redirect:
-                    self.ssh_cmd(
-                        "uptime", connect_timeout=5, redirect=redirect)
+                # Setting redirect=False allows the user to see errors like
+                # unix_listener: path "/tmp/rkn_ray_ssh_sockets/..." too long
+                # for Unix domain socket.
+                self.ssh_cmd("uptime", connect_timeout=5, redirect=False)
```

Comment:
The user will see some instances of `ssh: connect to host 34.221.9.122 port 22: Operation timed out`, but will also see important errors like the name too long.